### PR TITLE
Revised the scenario.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

**Comment by m1cm1c** [comment]
I'm closing this PR right away because there are conflicts.

**Comment by jGleitz** [comment]
You could really wait for #88!

**Comment by jGleitz** [comment]
(I made changes to the scenario, too)

**Comment by m1cm1c** [comment]
I only checked the revise issue which wasn't assigned when I started editing.

**Comment by jGleitz** [comment]
> I only checked the revise issue which wasn't assigned when I started editing.

That’s true. It was my fault not to assign myself to it, although I thought that nobody would start it before #88 was fixed.

If you merge your changes I’ll make my changes onto yours.

**Comment by m1cm1c** [comment]
Okay, I made this PR mergeable.

File: doc/Requirements Specification/Parts/Models.lyx [hunk 1]
```diff
@@ -142,19 +141,27 @@ EmmaSun
...
- find any bottlenecks that can be repaired with lesser effort.
+ as well as increasing waiting times in individual applications.
+ Unfortunately, the software system is based on an early design which has
+ grown over years with missing documentation in many cases.
+ Because of the enormous amount of effort it'd require, re-writing the entire
+ software system is an almost impossible task.
+ The only solution is to re-analyse the software system's source code and
+ hopefully find all bottlenecks which can be repaired with less effort.
  But re-analysing source code is also a quite unmanageable task.
+<<<<<<< HEAD
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: I think there went something wrong when merging!

File: doc/Requirements Specification/Parts/Models.lyx [hunk 2]
```diff
@@ -197,6 +205,10 @@ glspl{software architect}
 
 \end_inset
 
+<<<<<<< HEAD
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: I think there went something wrong when merging!

File: doc/Requirements Specification/Parts/Models.lyx [hunk 3]
```diff
@@ -277,6 +290,9 @@ glspl{SEFF}
 \end_inset
 
 .
+<<<<<<< HEAD
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: I think there went something wrong when merging!

**Comment by acBerger** [comment]
@m1cm1c please resolve conflicts

**Comment by jGleitz** [comment]
@m1cm1c, no, don’t. I”ll try to apply the changes you made here to my new version and create a PR for it. Could you revise the whole chapter again afterwards?

**Comment by m1cm1c** [comment]
Okay. Yes, I'll revise it again.

**Pull request closed without merging**